core/shared/src/main/scala/zio/Fiber.scala

```diff
@@ -285,11 +285,9 @@ object Fiber {
    */
   final def succeed[E, A](a: A): Fiber[E, A] = done(Exit.succeed(a))
 
-  /**
-   * Returns a fiber that is already succeeded with the specified lazily
-   * evaluated value.
-   */
-  final def succeedLazy[E, A](a: => A): Fiber[E, A] = done(Exit.succeed(a))
+  @deprecated("use succeed", "1.0.0")
```

Comment:
There is no `effectTotal` on `Fiber`, so I'm inclined to let this one stay since there's no alternative?.. /cc @jdegoes